src/components/FooterWrapper.js

```diff
@@ -93,7 +93,7 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
       displayName: 'Terra.bio'
     }),
     div({ style: { flexGrow: 1 } }),
-    div({ onClick: () => Nav.goToPath('hall-of-fame'), style: { fontWeight: 600, fontSize: '10px' } }, [
+    div({ style: { fontWeight: 600, fontSize: '10px' } }, [
```

Comment:
Looks the same:

![image](https://user-images.githubusercontent.com/484484/206574325-3c8a11bd-421a-4ed7-b076-654b18ebf8b5.png)